exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerBuilder.java

```diff
@@ -46,21 +46,20 @@ public PrometheusHttpServerBuilder setExecutor(ExecutorService executor) {
     return this;
   }
 
+  /** Sets the {@link PrometheusRegistry} to be used for {@link PrometheusHttpServer}. */
+  public PrometheusHttpServerBuilder setPrometheusRegistry(PrometheusRegistry prometheusRegistry) {
```

Comment:
@jkwatson you may be interested in this. May remove the need for [prometheus client bridge](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/prometheus-client-bridge).